src/Admin/Db/ArticleTableGateway.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Admin\Db;
+
+use Admin\Model\Entity\ArticleEntity;
+use Admin\Model\Storage\ArticleStorageInterface;
+use Zend\Db\TableGateway\TableGateway;
+use Zend\Db\Adapter\AdapterInterface;
+use Zend\Db\ResultSet\ResultSetInterface;
+
+class ArticleTableGateway extends TableGateway implements ArticleStorageInterface
+{
+    protected $columns = array(
+//        'articleUuid' => 'article_uuid',
+//        'title' => 'title',
+//        'publishedAt' => 'published_at',
+//        'createdAt' => 'created_at',
+//        'slug' => 'slug',
+//        'type' => 'type',
+    );
+
+    /**
+     * @param AdapterInterface   $adapter
+     * @param ResultSetInterface $resultSet
+     */
+    public function __construct(AdapterInterface $adapter, ResultSetInterface $resultSet)
+    {
+        parent::__construct('articles', $adapter, null, $resultSet);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function fetchAll($params)
+    {
+        return $this->select($params);
+
+//        $select = $this->getSql()->select();
+//
+//        return $this->selectWith($select);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function fetchOne($params)
+    {
+        $select = $this->getSql()->select();
```

Comment:
although this is a correct way, there is a short one 
`return $this->select(['article_uuid' =>  $params['article_uuid']])->current(); 
`